src/mono/netcore/System.Private.CoreLib/src/System/Threading/Thread.Browser.Mono.cs

```diff
@@ -1,10 +1,18 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Runtime.Versioning;
+
 namespace System.Threading
 {
     public partial class Thread
     {
         internal const bool IsThreadStartSupported = false;
+
+        [UnsupportedOSPlatform("browser")]
+        public void Start() => throw new PlatformNotSupportedException();
```

Comment:
Note that threads might be supported in the browser in the future.

Reply:
This does not change the status-quo only make it more visible to managed side

Reply:
this should probably replicate the `"Cannot start threads on this runtime."` message that we had in native.

Reply:
We don't do it for other PNSE and I think in this case it'll be quite obvious from the stacktrace